### DISCUSSION TITLE
change node configuration via mqtt

### DISCUSSION THREAD

**Discussion by hestiap** [zwave-js/zwave-js-ui#1960] [Q&A] [answered]
Hi,
that's probably already present in the Discussions, but I could not find something I could use, because I'm a beginner with mqtt :-(
I'd like to modify a value in the configuration of a node
[95-112-0-72] to 6 for example
I've found something like
`{ "args": [ "const {ConfigurationCCSet} = this.require('zwave-js');const cc = new ConfigurationCCSet(driver, {nodeId: [95], parameter: 72, value: 6, valueSize: 1}); await driver.sendCommand(cc);" ] }`
But I couldn't figure out how to use it.
I've found this
`Topic

<mqtt_prefix>/_EVENTS_/ZWAVE_GATEWAY-<mqtt_name>/<driver|node|controller>/<event_name>

Payload

{
  "data": [ ...eventArgs ] // an array containing all args in order
}`
But with no example, that's not very clear for me...
Do you have an example? or could you give me some hint please?

**Comment by AlCalzone** [comment]
@robertsLando I think this refers to your documentation :)

**Comment by robertsLando** [comment]
@hestiap You are actually using a driver function but why don't simply use a normal set value? 

https://zwave-js.github.io/zwavejs2mqtt/#/guide/mqtt?id=set-values

**Comment by hestiap** [comment]
I didn't know it could do it as it is a configuration modification
I'll try in this way, but I'd like to have a complete example as I'm also beginner with mqtt

**Comment by hestiap** [comment]
I've tried this
`mosquitto_pub -t zwave/nodeID_95/112/0/72/set -m "{"value": `6}"``
and got this
![Screenshot 2021-11-13 220339](https://user-images.githubusercontent.com/73014953/141659046-b4b144b4-79d8-4a11-900b-9ab2d6e468b6.png)
No change in the value :-(
![Screenshot 2021-11-13 220231](https://user-images.githubusercontent.com/73014953/141659049-67acba1c-ac0e-4462-888a-20704675a98c.png)

**Reply by robertsLando** [reply]
Could you check logs to see if the backend receives the set request? You should see something like `Write request received ...`

**Comment by hestiap** [comment]
Yes!
2021-11-15 20:09:51.319 INFO MQTT: Message received on zwave/nodeID_95/112/0/72/set: '{value: 6}'

**Reply by robertsLando** [reply]
@hestiap Ok that log should follow a write request log, if not means that for some reason there was no listener for that specific value. Also what z2m version are you running?

**Comment by hestiap** [comment, marked as answer]
@robertsLando
it was not the exact node in the message...
with this message `mosquitto_pub -t zwave/95/112/0/72/set -m 8` it is OK now 
Thanks for your help

**Reply by robertsLando** [reply]
You welcome :)
